apps/account-api/src/controllers/v1/delegation-v1.controller.ts

```diff
@@ -20,8 +20,8 @@ import {
 } from '@nestjs/common';
 import { ApiBody, ApiCreatedResponse, ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
 
-@Controller('v1/delegation')
-@ApiTags('v1/delegation')
+@Controller({ version: '1', path: 'delegation' })
```

Comment:
why is this version a string and the version in the delegation-v2.controller.ts an array?

Reply:
https://github.com/ProjectLibertyLabs/gateway/pull/513/files#r1765252182